src/lib.rs

```diff
@@ -266,12 +266,18 @@ impl StreamDeck {
                 states.extend(vec![0u8; (self.kind.key_count() + 1) as usize]);
 
                 if data[9] != 0 {
-                    let key = if self.kind == Kind::Akp815 {
-                        inverse_key_index(&self.kind, data[9] - 1)
-                    } else {
-                        ajazz153_to_elgato_input(&self.kind, data[9] - 1)
+                    let key = match self.kind {
+                        Kind::Akp815 => inverse_key_index(&self.kind, data[9] - 1),
+                        Kind::Akp153 | Kind::Akp153E | Kind::Akp153R | Kind::MiraBoxHSV293S => ajazz153_to_elgato_input(&self.kind, data[9] - 1),
+                        _ => data[9] - 1
```

Comment:
I think this should be an explicit check for your device instead of a fall through case to prevent any issues when someone else adds another v1 device. If Rust doesn't let you do this without a fallback case make it `_ => unimplemented!()`

Reply:
Yes, and change the last one to `_ => unimplemented!()`, or remove it entirely if Rust lets you.